doc/man3/OSSL_PARAM.pod

```diff
@@ -160,6 +160,7 @@ systems.
 =item B<OSSL_PARAM_REAL>
 
 The parameter data is a floating point value in native form.
+Currently only the C<double> type is supported.
```

Comment:
I'd add "by OpenSSL and it's providers" or some such.  A third party could use float without our helpers and it would work between their application and their provider.  Pretty rare situation I suspect.

Reply:
> A third party could use float

I'd rather not open that can of worms, unless you feel strongly about it.

Reply:
Not strongly enough to have a disagreement over it.